MapboxNavigation/RouteTableViewController.swift

```diff
@@ -42,6 +42,8 @@ class RouteTableViewController: UIViewController {
             headerView.distanceRemaining.text = distanceFormatter.string(from: routeProgress.distanceRemaining)
         }
         
+        dateComponentsFormatter.unitsStyle = routeProgress.durationRemaining < 3600 ? .short : .abbreviated
```

Comment:
I think `dateComponentsFormatter` rounds up when the duration is close to an hour. So there might be an edge case where it still uses `short` even though you want `abbreviated`.